hpi-schul-cloud/nuxt-client
# BC-4913 remove passage on user registration

# Short Description

QA note:
change tested on env: 
[https://nbc-bc-4913-remove-passage-on-user-registration.cd.dbildungscloud.dev/](https://brb-bc-4913-remove-passage-on-user-registration.cd.dbildungscloud.dev/)
[https://default-bc-4913-remove-passage-on-user-registration.cd.dbildungscloud.dev/](https://brb-bc-4913-remove-passage-on-user-registration.cd.dbildungscloud.dev/)
https://brb-bc-4913-remove-passage-on-user-registration.cd.dbildungscloud.dev/
No issues found. Text for new admin and old admin page are removed.
See Jira ticket: https://ticketsystem.dbildungscloud.de/browse/BC-4913





## Links to Ticket and related Pull-Requests



- https://ticketsystem.dbildungscloud.de/browse/BC-4913
- https://github.com/hpi-schul-cloud/schulcloud-client/pull/3293


## Changes



## Data-security



## Deployment



## New Repos, NPM packages or vendor scripts



## Screenshots of UI changes



## Checklist before merging


> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed
